.github/workflows/yaml-lint.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: GH Action YAML linter
+
+on:
+  pull_request:
+    paths:
+      - ".github/**/**.yml"
+      - ".github/**/**.yaml"
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  main:
+    name: YAML Lint
+    runs-on: ubuntu-20.04
+    steps:
+      - name: GIT | Checkout source code
+        uses: actions/checkout@v3
+        with:
+          repository: Cardinal-Cryptography/aleph-node
+          token: ${{ secrets.CI_GH_TOKEN }}
+
+      - name: GIT | Check diff
+        id: check-diff
+        run: |
+          git fetch --quiet origin ${GITHUB_BASE_REF}:refs/remotes/origin/${GITHUB_BASE_REF}
+          YAML_FILES=`git --no-pager diff origin/${GITHUB_BASE_REF} --name-only \
+          | grep -E ".github\/.*\/.*(.yaml|.yml)" | tr '\n' ' '`
+          echo "Changed YAML files: ${YAML_FILES}"
+          echo "YAML_FILES=`echo $YAML_FILES`" >> $GITHUB_ENV
+
+      - name: LINT | Execute yaml-lint
+        uses: ibiqlik/action-yamllint@v3.1.1
```

Comment:
Does this action use yamllint version `1.18`? https://github.com/nosborn/github-action-yamllint/blob/master/Dockerfile#L5

As the newest one is `1.29`, those GH actions written by someone else seems like not the way to go in the long term. But this is a general comment not related to this PR.

Reply:
This action uses `yamllint>=1.25.0`
https://github.com/ibiqlik/action-yamllint/blob/v3.1.1/Dockerfile#L4

This has nothing in common to action you sent because it is totally different repo (which is archived btw) made by someone else. :smile: